### PR TITLE
Rework configuring hosts using Salt

### PR DESCRIPTION
**Pull request by maximiliankolb**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)

### CODE REVIEW

File: guides/common/modules/con_introduction-to-salt.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+====
+Salt offers two distinct modes of operation:
+Clientless using SSH or the Salt Minion client software.
+
+ifdef::orcharhino[]
+{Project}'s Salt plug-in supports exclusively the Salt Minion approach.
+endif::[]
+====
+
+.Additional Resources
```
Comment (maximiliankolb) [inline guides/common/modules/con_introduction-to-salt.adoc:19]: Is this the place we were talking about on Monday? cc @asteflova

File: guides/common/modules/con_introduction-to-salt.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+This guide describes how to use Salt for configuration management in {Project}.
+This guide contains information about how to install the Salt plug-in, how to integrate {Project} with an existing Salt Master, and how to configure managed hosts with Salt.
+
+[NOTE]
+====
+Salt offers two distinct modes of operation:
+Clientless using SSH or the Salt Minion client software.
+
+ifdef::orcharhino[]
+{Project}'s Salt plug-in supports exclusively the Salt Minion approach.
```
Comment (adamlazik1) [inline guides/common/modules/con_introduction-to-salt.adoc:15]:
```suggestion
Salt plug-in in {Project} supports exclusively the Salt Minion approach.
```

File: guides/common/modules/proc_activating-salt.adoc [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+----
+# systemctl restart salt-master salt-api
+----
+. On your {ProjectServer}, restart all {Project} services:
++
+[options="nowrap" subs="attributes"]
+----
+# {foreman-maintain} service restart
+----
+. In the {ProjectWebUI}, navigate to *Infrastructure > {SmartProxies}*.
```
Comment (adamlazik1) [inline guides/common/modules/proc_activating-salt.adoc:19]:
```suggestion
. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*.
```

**Review by adamlazik1 — COMMENTED** [review]
Two minor notes, rest LGTM

File: guides/common/modules/proc_activating-salt.adoc (hunk 3 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_activating-salt.adoc]: Applied.

**Review by adamlazik1 — APPROVED** [review]

File: guides/common/modules/con_introduction-to-salt.adoc (hunk 1 above)
Comment (Lennonka) [reply guides/common/modules/con_introduction-to-salt.adoc:19]: Yes, that's it.

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: :heavy_check_mark: 
   643371af0..c72a5f5ea  3.8 -> 3.8
   7af7e4204..47ff175a0  3.7 -> 3.7

**Pull request merged**